pkg/mod/module.go

```diff
@@ -8,18 +8,32 @@ import (
 	"github.com/pkg/errors"
 )
 
-var SyslModules = os.Getenv("SYSL_MODULES") != "off"
+var SyslModules = os.Getenv("SYSL_MODULES") != SyslModulesOff
+var GitHubMode = os.Getenv("SYSL_MODULES") == SyslModulesGitHub
+
+const (
+	SyslModulesOff    = "off"
+	SyslModulesOn     = "on"
+	SyslModulesGitHub = "github"
+	MasterBranch      = "master"
+)
 
 type Module struct {
-	Name    string
-	Dir     string
-	Version string
+	Name    string // "github.com/anz-bank/sysl"
+	Dir     string // "/Users/foo/go/pkg/mod/github.com/anz-bank/sysl@v1.1.0"
+	Version string // "v1.1.0"
 }
 
 type Modules []*Module
 
 var modules Modules
 
+type DependencyManager interface {
+	Get(filename, ver string, m *Modules) (*Module, error)
```

Comment:
Some comments would be super nice here, to help distinguish between the Get and Find interface. Looks to me like they're the same except for the returned error?